### ISSUE TITLE
CheckBox and Radio Button colors

### ISSUE THREAD

**Issue by fpabl0** [fyne-io/fyne#1855]
### Is your feature request related to a problem? Please describe:

No, it isn't. It is just a style opinion.

### Is it possible to construct a solution with the existing API?

Yes, it is.

### Describe the solution you'd like to see:

When using the fyne_demo, there are some tabs that have CheckBoxes and RadioButtons and they seem to need more color. Could these widgets have the primary color when selected? Something like this:

<img width="484" alt="Captura de Pantalla 2021-01-23 a la(s) 21 59 47" src="https://user-images.githubusercontent.com/12239342/105620070-00e7d300-5dc7-11eb-9afe-fc0ea0ec3920.png">

**Comment by AlbinoGeek** [comment]
I think they are meant to have the `ActiveColour`, which is a shade of the PrimaryColour (we just don't have this yet.)

**Comment by fpabl0** [comment]
Yeah, maybe, although looking at Material Design Specs, the ActiveColor seems to be the same Primary color:

<img width="363" alt="Captura de Pantalla 2021-01-24 a la(s) 00 58 07" src="https://user-images.githubusercontent.com/12239342/105622398-6d6ecc00-5ddf-11eb-86cd-de12c3eb9435.png">

However, I think the major problem is the background color as it has different states (and colors):

<img width="400" alt="Captura de Pantalla 2021-01-24 a la(s) 00 58 07" src="https://lh3.googleusercontent.com/uIZAD8BqdUty_SKq4IqRBcQ5LZEDl_4obrO2eWArEW2FNrP6o1PgRAJrmaGM1nfF9pCI7dQKvftHIJHFBSFpAS_1HbT_3RPSv8DrJjI=w1064-v0">

**Comment by AlbinoGeek** [comment]
The hover circle was JUST added in 2.0 to radio buttons-- and it doesn't appear to be animated yet (that looks like an opacity animation for the bottom two rows.)

Good find.

**Comment by andydotxyz** [comment]
The background circles have been with us for some time. They have hover and focus states, but not yet added active.
